### PR TITLE
Bump decidim from v0.27.3 to v0.28.0.rc4

### PR DESCRIPTION
**Pull request by andreslucena**
This PR upgrades Metadecidim to the last release candidate available, v0.28.0.rc4

## Testing

Everything should be green

## Screenshot

Screenshot of the new homepage

### CODE REVIEW

File: config/initializers/decidim.rb [hunk 1]
```diff
@@ -646,3 +646,6 @@
     }
   ]
 end
+
+# Workaround until v0.28.0 or v0.28.0.rc3 is available
```
Comment (alecslupu) [inline config/initializers/decidim.rb:650]: If i remember correctly, the fix will be in v0.28.0.rc4 ?

File: package.json [hunk 2]
```diff
@@ -2,27 +2,23 @@
   "name": "metadecidim",
   "private": true,
   "dependencies": {
-    "@decidim/browserslist-config": "^0.27.3",
-    "@decidim/core": "^0.27.3",
-    "@decidim/elections": "^0.27.3",
-    "@decidim/webpacker": "^0.27.3"
+    "@decidim/browserslist-config": "~0.28.0-rc2",
```
Comment (alecslupu) [inline package.json:5]: maybe we need to update the versions to "~0.28.0-rc3" ?

**Review by alecslupu — CHANGES_REQUESTED** [review]
Can you have a look on the package.json file ?

File: config/initializers/decidim.rb (hunk 1 above)
Comment (andreslucena) [reply config/initializers/decidim.rb:650]: Nop, it was in rc3. This workaround is no longer needed, nice catch 👏🏽 👏🏽

File: package.json (hunk 2 above)
Comment (andreslucena) [reply package.json]: Done in 5b7c67d3642443bc4a1e9a1ab385b534d36e5d2e

**Review by fblupi — APPROVED** [review]
LGTM! Tested locally and it works fine 🚀

**Review by fblupi — APPROVED** [review]

**Review by alecslupu — APPROVED** [review]
:+1:

**Pull request merged**